pymc3/distributions/__init__.py

```diff
@@ -345,11 +354,17 @@ def transform_logp(logp_var: TensorVariable, inputs: List[TensorVariable]) -> Te
     for measure_var in inputs:
 
         transform = getattr(measure_var.tag, "transform", None)
+        rv_var = getattr(measure_var.tag, "rv_var", lambda: None)()
+
+        if transform is not None and rv_var is None:
+            warnings.warn(
+                f"A transform was found for {measure_var}" " but no corresponding random variable"
```

Comment:
String is a bit messed up.
More importantly: The sentence is a bit incomplete - no variable corresponding to what?

Reply:
That `measure_var` doesn't have a random variable associated with it, so there's really nothing else to print or say.  If anything, this should probably be an error condition.

Reply:
It might actually make more sense to associate the transform object with the `rv_var` (i.e. the random variable).  I'll have to think about that.